api/tsconfig.json

```diff
@@ -5,5 +5,12 @@
     "strictNullChecks": true,
     "alwaysStrict": true,
     "sourceMap": true
-  }
+  },
+  "lib": [
+    "DOM",
+    "ES6",
+    "DOM.Iterable",
+    "ScriptHost",
+    "ES5.Promise"
```

Comment:
Oh, this was a thing I sort of glossed over when I made the original PR for this editor--I had assumed that staff/admins would be running a pretty recent version of a modern browser, so I didn't include much/anything in the way of polyfills or even transpilation, IIRC. Should we file an issue to address that later?

Reply:
Good point! Creates #872